frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/GitMacheteRepositoryUpdateTask.java

```diff
@@ -86,9 +87,9 @@ public void run(ProgressIndicator indicator) {
    * This method is heavyweight and must never be invoked on UI thread.
    */
   private Option<IGitMacheteRepository> updateRepository() {
-    Path mainDirectoryPath = getMainDirectoryPath(gitRepository);
-    Path gitDirectoryPath = getGitDirectoryPath(gitRepository);
-    Path macheteFilePath = getMacheteFilePath(gitRepository);
+    Path mainDirectoryPath = Paths.get(getMainDirectory(gitRepository).getPath());
```

Comment:
Hmm maybe there should be a helper that returns a `Path` in `GitVfsUtils`? As it used to be in `GitPathUtils`

Reply:
Esp. that there's `resolveMacheteFilePath`